Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -614,64 +681,32 @@ bool UserCmd_Process(uint client, const wstring &wscCmd)
 	return false;
 }
 
-void __stdcall JumpInComplete(uint iSystemID, uint iShip)

```

Comment:
so wait how do we get away without this one?

Reply:
It compares current systems from PlayersDB.

Reply:
To me it seems that we don't actually keep it up to date all the time though? Is that okay?

Reply:
Tests show that it is okay.